src/main/java/org/testng/internal/Invoker.java

```diff
@@ -221,7 +226,7 @@ private void invokeConfigurations(IClass testClass,
         } else {
           invokeConfigurationMethod(newInstance, tm, parameters, testResult);
         }
-
+        testResult.setStatus(ITestResult.SUCCESS);
```

Comment:
Could you check you are not overriding status from `MethodInvocationHelper`?

Reply:
@juherr - No I am not overriding the status. I have checked it. Thanks to you, i realized I can actually get rid of this. So I amended the commit, reverted this changeset and push in the changes. Please check